config.docker.register.yaml

```diff
@@ -33,9 +32,8 @@ externalConfigDirectory: /tmp
 downloadConfigs: true
 
 # can be http: file s3: or classpath: url
-fieldsYamlLocation: classpath://config/fields.yaml
-
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: http://field/records.yaml?page-size=5000
```

Comment:
The point of this thing is that people can spin up and experiment with the whole stack--we want people to have these registers running locally. I'm only pre-loading them with data to make the initial steps less painful.